metadata/src/main/java/org/apache/kafka/controller/errors/EventHandlerExceptionInfo.java

```diff
@@ -88,11 +86,6 @@ public static EventHandlerExceptionInfo fromInternal(
             return new EventHandlerExceptionInfo(false, false, internal,
                 new PolicyViolationException("Unable to perform excessively large batch " +
                     "operation."));
-        } else if (internal instanceof UnexpectedBaseOffsetException) {
```

Comment:
So this will be treated as an UnknownServerException if it does happen? I guess that's reasonable

Reply:
I actually removed this exception. KRaft is not checking the base offset anymore so it doesn't throw this exception anymore.